src/table/settings.ini

```diff
@@ -938,12 +938,12 @@ type     = SLE_UINT8
 from     = SLV_87
 guiflags = SGF_MULTISTRING
 def      = 2
-min      = 0
+min      = 1
```

Comment:
I think it is the max value that should be changed.
NPF becomes 0
YAPF becomes 1